src/applications/check-in/locales/tl/translation.json

```diff
@@ -0,0 +1,209 @@
+  "you-can-check-in-starting-at-this-time": "Maaari kang mag-check in simula sa oras na ito: {{date, time}}",
+  "you-can-pre-check-in-online-before-midnight-of-the-day-of-your-appointment": "Maaari kang mag-pre-check-in online bago maghatinggabi sa araw ng iyong appointment.",
+  "you-can-pre-check-in-online-until-date": "Maaari kang mag-pre-check in online hanggang sa {{date, mdY}}.",
+  "you-can-sign-in-to-your-va-account-to-update-your-contact-information-online": "Maaari kang mag-sign in sa iyong VA account para i-update ang iyong impormasyon sa pakikipag-ugnayan online.",
+  "you-can-still-check-in-with-your-phone-on-the-day-of-your-appointment": "Maaari ka pa ring mag-check-in sa iyong phone sa araw ng iyong appointment.",
+  "you-can-still-check-in-with-your-phone-once-you-arrive-at-your-appointment": "Maaari ka pa ring mag-check-in gamit ang iyong phone sa oras na dumating ka sa iyong appointment.",
+  "you-checked-in-at": "Nag-check in ka noong {{date, time}}",
+  "you-may-need-to-wait-about-15-minutes-for-their-call-thanks-for-your-patience": "Maaaring kailanganin mong maghintay ng humigit-kumulang 15 minuto para sa kanilang tawag. Salamat sa iyong pagtitiyaga.",
+  "your-answers-will-help-us-better-prepare-for-your-needs": "Ang iyong mga sagot ay makakatulong sa aming mas mapaghandaan ang iyong mga pangangailangan.",
+  "your-appointment-started-more-than-15-minutes-ago-we-cant-check-you-in-online-ask-a-staff-member-for-help": "Nagsimula ang iyong appointment ng mahigit 15 minuto ang nakakaraan. Hindi ka namin mai-check in online. Humingi ng tulong sa isang staff member.",
+  "your-appointments": "Ang iyong mga appointment",
+  "your-appointments-on-day-facility_one": "Ang iyong appointment ay sa {{day, long}} sa {{facility}}.",
+  "your-appointments-on-day-facility_other": "Ang iyong mga appointment ay sa {{day, long}} sa {{facility}}.",
+  "your-date-of-birth-can-not-be-in-the-future": "Ang iyong petsa ng kapanganakan ay hindi maaaring nasa hinaharap",
+  "your-last-name": "Ang iyong apelyido",
+  "your-provider-will-call-you": "Tatawagan ka ng iyong provider.",
+  "your-provider-will-call-you-at-your-appointment-time-you-may-need-to-wait-about-15-minutes-for-their-call-thanks-for-your-patience": "Tatawagan ka ng iyong provider sa oras ng iyong appointment. Maaaring kailanganin mong maghintay ng humigit-kumulang 15 minuto para sa kanilang tawag. Salamat sa iyong pagtitiyaga.",
+  "youre-also-responsible-for-protecting-your-personal-health-information-if-you-print-or-download-your-information-or-share-it-electronically-with-others-youll-need-to-take-steps-to-protect-it": "Ikaw rin ay responsible sa pagprotekta sa iyong personal na impormasyong pangkalusugan. Kung ipi-print o ida-download mo ang iyong impormasyon—o ise-share sa iba sa elektronikong paraan—kakailanganin mong gumawa ng mga hakbang para maprotektohan ito.",
+  "youre-checked-in-for-your-appointment": "Naka-check in ka na para sa iyong appointment sa {{date, time}}",
```

Comment:
This key isn't being used any more and the confirmation title is defaulting to English. More of these could be outdated.